Makefile

```diff
@@ -1,5 +1,10 @@
 .PHONY: start start_dev
```

Comment:
You should add `build` to the list here, as running the target/recipe doesn't create a file called `build`.